### ISSUE TITLE
Segmentation Fault

### ISSUE THREAD

**Issue by MdAsifKhan** [thunlp/TensorFlow-TransX#3]
Hi I'm trying this tensorflow implementation of trans methods, I get "Segmentation Fault" error each time I try to run any of the script.

**Comment by THUCSTHanxu13** [comment]
If your tensorflow version is r1.0 or higher, you may change some API interfaces in the program. We build our project based on tensorflow r0.11. Another reason for your "Segmentation Fault" is that your data format may be different from ours, you can check the format of your dataset. '''string inPath = "./data/" ''' can let the program know where is your dataset, make sure the route is right. If none of these reasons for your program, you can email me more detail information and I will help you solve the problem.

**Comment by MdAsifKhan** [comment]
thanks.!, the issue was with data format works fine now. I'm trying to do it for some different knowledge graph. The loss is converging but still seems to be very high around 1000. Would be good to use tensorboard for plotting training and validation loss.

**Comment by samithaj** [comment]
what dataset did u use ?
one from here:http://pan.baidu.com/s/1eRD9B4A still give me a segmentation fault

**Comment by MdAsifKhan** [comment]
I did experiment with freebase it works fine. Make sure you have data in the mentioned format.
